.github/workflows/ci.yml

```diff
@@ -54,12 +56,70 @@ jobs:
     - name: Upload coverage to Codecov
       uses: codecov/codecov-action@v1
 
+  # API Definition testing
+  # https://docsapi.io/cli
+  api_definition:
+    name: API Definition Tests
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:12
+        ports: ['5432:5432']
+        env:
+          POSTGRES_PASSWORD: postgres
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    strategy:
+      matrix:
+        otp: ['25.1.2']
+        elixir: ['1.14.2']
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Elixir
+      uses: erlef/setup-beam@v1
```

Comment:
Setting up `Elixir` _twice_ seems resource-wasteful ... 💭 
Is this duplication _needed_? 🤷‍♂️

Reply:
I did this to run both jobs concurrently and keep concerns separated.
That way we will know in Github Action's dashboard if the API definition tests failed or the build failed.

Reply:
The difference 

![image](https://user-images.githubusercontent.com/17494745/213881603-3e2cd552-71ec-4f81-a56a-3198507a8db5.png)
![image](https://user-images.githubusercontent.com/17494745/213881607-f8dfd71a-b52e-46c8-a27b-cc6b87a5d32b.png)

Reply:
Totally understood. 
Perhaps my question was unclear/incomplete.
I wanted to understand why we're doing the setup _twice_ instead of re-using the steps and avoiding duplication in the `ci.yml` file. This is fine on the surface _initially_ but incurs _maintenance_ ... 

Leaving it like this for now. Will revisit later.

Reply:
By far the slowest part of the CI is deploying the review app:

<img width="852" alt="image" src="https://user-images.githubusercontent.com/194400/213882483-08a0f0f3-ae45-47cc-8ee4-401d321aed3c.png">

**`3 minutes`** to deploy ... ⏳